great_expectations/data_asset/util.py

```diff
@@ -127,6 +128,8 @@ def recursively_convert_to_json_serializable(test_obj):
     elif isinstance(test_obj, dict):
         new_dict = {}
         for key in test_obj:
+            if key == "row_condition":
```

Comment:
the verification is only done for row_condition field and not for the other.
so we hopefully don't disturb the other features